model/histogram/generic.go

```diff
@@ -540,3 +540,87 @@ var exponentialBounds = [][]float64{
 		0.9892280131939752, 0.9919100824251095, 0.9945994234836328, 0.9972960560854698,
 	},
 }
+
+// reduceResolution reduce the input spans, buckets in origin schema to the spans, buckets in target schema.
```

Comment:
reduce → reduces

Also mention here that the target schema must be smaller than the original schema. And explain the meaning of the `deltaBuckets` bool.